dbx_Pg.pl

```diff
@@ -56,6 +56,47 @@
 				"CREATE INDEX rptrecord_idx_serial6 ON rptrecord (serial, ip6);",
 				],
 			},
+		"tls_report" => {
+			column_definitions 		=> [
+				"serial"					, "bigint"									, "GENERATED ALWAYS AS IDENTITY",
+				"mindate"				, "timestamp without time zone"		, "NOT NULL",
+				"maxdate"				, "timestamp without time zone"		, "NULL",
+				"domain"					, "character varying(255)"				, "NULL",
+				"org"						, "character varying(255)"				, "NOT NULL",
+				"reportid"				, "character varying(255)"				, "NOT NULL",
+				"contact"				, "character varying(255)"				, "NULL",
+				"policy_type"			, "character varying(255)"				, "NULL",
+				"policy_string"		, "character varying(255)"				, "NULL",
+				"summary_failure"		, "bigint"									, "GENERATED ALWAYS AS IDENTITY",
+				"summary_successful"	, "bigint"									, "GENERATED ALWAYS AS IDENTITY",
+				"raw_json"				, "text"										, "",
+				],
+			additional_definitions 		=> "PRIMARY KEY (serial), UNIQUE KEY domain (domain,reportid)",
+			table_options			=> "ROW_FORMAT=COMPRESSED",
```

Comment:
This is a MySQL-specific syntax, so just make it
```suggestion
			table_options			=> "",
```